pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -275,3 +275,37 @@ func validateInitialScale(config *autoscalerconfig.Config, m map[string]string)
 	}
 	return nil
 }
+
+func validateMaxInitialScale(config *autoscalerconfig.Config, m map[string]string) *apis.FieldError{
```

Comment:
Is this function wired in anywhere? I'm not sure it's being invoked. Can you add a test